src/compiler/transformers/transform-utils.ts

```diff
@@ -616,11 +613,12 @@ export const getComponentTagName = (staticMembers: ts.ClassElement[]) => {
   return null;
 };
 
-export const isStaticGetter = (member: ts.ClassElement) => {
+export const isStaticGetter = (member: ts.ClassElement): boolean => {
   return (
-    member.kind === ts.SyntaxKind.GetAccessor &&
-    member.modifiers &&
-    member.modifiers.some(({ kind }) => kind === ts.SyntaxKind.StaticKeyword)
+    (member.kind === ts.SyntaxKind.GetAccessor &&
+      Array.isArray(retrieveTsModifiers(member)) &&
+      retrieveTsModifiers(member)?.some(({ kind }) => kind === ts.SyntaxKind.StaticKeyword)) ??
+    false
```

Comment:
Can't this be condensed so we don't need two calls to `retrieveTsModifiers`?
```suggestion
    (member.kind === ts.SyntaxKind.GetAccessor &&
      retrieveTsModifiers(member)?.some(({ kind }) => kind === ts.SyntaxKind.StaticKeyword)) ??
    false
```

Reply:
I went back and forth here. But I think coalescing them is the right call. Fixed in 96fcdcf89950830d31d0be4478e43dba6f82a2d0